src/Command/SelfUpdateCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace SensioLabs\Deptrac\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class SelfUpdateCommand extends Command
+{
+    public function configure()
+    {
+        $this
+            ->setName('self-update')
+            ->setAliases(array('selfupdate'))
+            ->setDescription('Updates deptrac.phar to the latest version.')
+        ;
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $latest = trim(file_get_contents('http://get.sensiolabs.de/deptrac.version'));
+        if (empty($latest)) {
+            $output->writeln('<error>Latest deptrac version info could not be fetched.</error>');
+
+            return 1;
+        }
+
+        if ($this->getApplication()->getVersion() === $latest) {
```

Comment:
i think we should keep it as simple as possible. from my point of view for the user it's just important, that he has the newest version.